packages/expo-av/ios/EXAV/EXAV.m

```diff
@@ -424,18 +436,12 @@ - (void)handleMediaServicesReset:(NSNotification *)notification
 {
   // See here: https://developer.apple.com/library/content/qa/qa1749/_index.html
   // (this is an unlikely notification to receive, but best practices suggests that we catch it just in case)
-  
-  _currentAudioSessionMode = EXAVAudioSessionModeInactive;
-  
-  [self _runBlockForAllAVObjects:^(NSObject<EXAVObject> *exAVObject) {
-    [exAVObject handleMediaServicesReset:nil];
-  }];
-  
-  if (_audioRecorder) {
-    [self _removeAudioRecorder:NO];
-    [self _createNewAudioRecorder];
```

Comment:
I had to make a judgment call here—I found that this method is called when a user is recording via airpods and then disconnects them by turning them off (or if the battery dies). If this happens, I don't want to tear down the entire audio session, because the user may want to continue recording using a different audio source. I didn't find any issues in retaining the current audio session in this case.